homeassistant/components/radarr/coordinator.py

```diff
@@ -75,6 +75,17 @@ async def _fetch_data(self) -> list[RootFolder]:
         return cast(list, await self.api_client.async_get_root_folders())
 
 
+class HealthDataUpdateCoordinator(RadarrDataUpdateCoordinator):
+    """Health update coordinator."""
+
+    async def _fetch_data(self) -> bool:
+        """Fetch the movies data."""
+        return any(
+            report.source in HEALTH_ISSUES
+            for report in await self.api_client.async_get_failed_health_checks()
+        )
+
```

Comment:
I feel like it make more sense to keep the list in the coordinator and do the any check in the entity onstead.